docs/source/dev_guide/release.rst

```diff
@@ -3,176 +3,83 @@ How to Prepare a Release
 
 In this guide, we'll cover:
 
-* Preparing The Code For Release
-* Creating A Release On GitHub
-* Updating The sha256
+* Bumping the Version
+* Releasing On GitHub
 * Releasing The Software On Anaconda
 * Creating a New Version of the Documentation
 
+Bumping the Version
+-------------------
 
-Preparing The Code For Release
-------------------------------
-
-These steps entail modifying files before a release is made.
-
-1. Pull the lastest code from whatever branch you want to release from.
-It's usually ``master``.
-
-    ::
-
```

Comment:
These commands should be excluded because the direct project contributor should be familiar with their git environment at this point (forking or pulling). Less doc upkeep for us.